### ISSUE TITLE
tensorflow版本？？

### ISSUE THREAD

**Issue by zhufeijuanjuan** [guotong1988/text-style-transfer-chinese#3]
在tf1.14下报错，input 0 of layer gru_cell_2 is incompatible with the layer: its rank is undefined, but the layer requires a defined rank

**Comment by zhufeijuanjuan** [comment]
> 在tf1.14下报错，input 0 of layer gru_cell_2 is incompatible with the layer: its rank is undefined, but the layer requires a defined rank

i solved it by modify:
self.dropout = tf.placeholder(tf.float32,name="dropout")
to 
self.dropout = tf.placeholder(tf.float32,shape=(),name="dropout")
in main.py
the same modification from self.learning_rate to self.batch_size
